### ISSUE TITLE
Option to list duplicate results by something other than file size

### ISSUE THREAD

**Issue by newadventure079** [theophanemayaud/video-simili-duplicate-cleaner#103]
It would be nice to have the option to list the duplicate results by folder name, and then by file name.  I assume other sort options could be included as well, such as creation date, etc

When the results are sorted by size, the folders and filenames are not in alphabetical order.  Having them in alphabetical order will allow us to more efficiently cull our duplicates because we know we're first working with the set of video from our `Amazon` trip, then next our `Bahamas` trip, and then our `Cancun` trip, so on.

**Comment by theophanemayaud** [comment]
Yup good idea, I sorted by size such that bigger duplicates would be treated first but it should be pretty straightforward to add options. 

- by folder and file name : folder and filename would be the hole path to the file, not just the first level folder above the file. Right ?

- by date should also be easy
